cmd/operator/README.md

```diff
@@ -16,6 +17,12 @@ administrators.
 
 ## Installation
 
+Operator deployment depends on NFD and cert-manager. Those components have to be installed to the cluster before the operator can be deployed.
```

Comment:
This sounds a bit misleading. They are not hard dependencies but the steps below guide how to install them because they are needed in the "default" setting.

Reply:
Sure. I'll reformulate it so that the default setup needs these components.